CGCharter-1727386911.html

```diff
@@ -250,25 +250,25 @@ <h2 id="decision">
       To afford asynchronous decisions and organizational deliberation, any
       resolution (including publication decisions) taken in a face-to-face
       meeting or teleconference will be considered provisional.
+      A <a href="#calls-for-consensus">call for consensus</a> (CfC) will be issued for all resolutions.
+    </p>
+    <h3 id="calls-for-consensus">
+      Calls for Consensus
+    </h3>
+    <p>
       A call for consensus (CfC) will be issued for all resolutions via
-      email to public-swicg@w3.org, preferably with the abbreviation CfC early
-      in the subject line to catch more attention.
+      email to public-swicg@w3.org. The presence of formal resolutions will be indicated by a "CfC" prefix in the subject line of an email on the list. Additional outreach to community venues for more affirmative consent is strongly encouraged.
```

Comment:
It's best to maintain the approximate line length.
```suggestion
      email to public-swicg@w3.org. The presence of formal resolutions will be
      indicated by a "CfC" prefix in the subject line of an email on the list.
      Additional outreach to community venues for more affirmative consent is
      strongly encouraged.
```